metadata.json

```diff
@@ -22,43 +29,23 @@
       "version_requirement": ">= 2.0.0 < 4.0.0"
     }
   ],
-  "description": "Install/configure MySQL with Galera/XtraDB cluster support",
-  "license": "BSD-2-Clause",
-  "name": "fraenki-galera",
   "operatingsystem_support": [
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "8",
-        "9"
+        "8"
```

Comment:
Why was support for Debian 9 removed?

Reply:
Because the tests for those were failing. I explained that in the commit message. I don't expect this MR to be merged as is.
But re-enabling the tests show some issues that have to be afdressed.

Reply:
Fixed and even added debian 10 (buster)